Core/Core/UIViews/GradeCircleView.swift

```diff
@@ -63,8 +63,17 @@ public class GradeCircleView: UIView {
         loadFromXib()
     }
 
-    public func update(_ assignment: Assignment) {
+    /**
+     - parameters:
+       - circleColor: The color of the grade circle. If not specified it will use the default color specified in `CircleProgressView`.
+     */
+    public func update(_ assignment: Assignment, circleColor: UIColor? = nil) {
         gradeCircle.progress = 1 // make sure it's never spinning
+
+        if let circleColor = circleColor {
+            gradeCircle.color = circleColor
```

Comment:
If you set `gradeCircle.color = nil`, it will use the inherited `tintColor`. But this doesn't seem to invasive either, so I'm good either way.